### DISCUSSION TITLE
Why was v1.15.2712.0 not uploaded to github?

### DISCUSSION THREAD

**Discussion by ElitePheonix2009** [microsoft/terminal#14405] [Q&A] [answered]
Pls tell Why was v1.15.2712.0 not uploaded to GitHub?

**Comment by zadjii-msft** [comment, marked as answer]
https://github.com/microsoft/terminal/releases/tag/v1.15.2712.0? That's just the version that shipped with the OS, so it's got some different signing going on. It was immediately superceeded by some bugfixes in [v1.15.287](https://github.com/microsoft/terminal/discussions/14250#discussion-4484690), which we were able to find between (checking v1.15.2712.0 into the OS codebase) and (the version of Windows containing v1.15.2712.0 to actually ship to customers).

No sense in bothering running an additional CI and release pipeline just to bookmark a version of the code that had some immediate bugfixes.

**Comment by DHowett** [comment]
Ah, the real reason is that all the fixes were merged into the next version and I didn't want to send an e-mail to every subscriber about an intermediate version that should have been replaced by the store automatically. :smile:

**Reply by DHowett** [reply]
(Why do you ask? Are you stuck on that version? How did you get it, but not 1.15.2852?)

**Reply by ElitePheonix2009** [reply]
> (Why do you ask? Are you stuck on that version? How did you get it, but not 1.15.2852?)

I have got 1.15.2852, I was just asking because of curiosity.
